src/main/webapp/WEB-INF/views/car/cars.jsp

```diff
@@ -0,0 +1,58 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title style="text-align: center">Displaying all Cars in our service</title>
+</head>
+<body>
+<h1 style="color: chocolate;font-size: 50px; text-align: center;">
+    List of cars:
+</h1>
+<table style="border:1px solid black;margin-left:auto;margin-right:auto;">
+    <tr style="font-size: 30px;">
+        <td style="text-align:center; padding: 15px">
+            <b>ID</b>
+        </td>
+        <td style="text-align:center; padding: 15px">
+            <b>MODEL</b>
+        </td>
+        <td style="text-align:center; padding: 15px">
+            <b>MANUFACTURER</b>
+        </td>
+        <td style="text-align:center; padding: 15px">
+            <b>DRIVERS</b>
+        </td>
+        <td style="text-align:center; padding: 15px">
+            <b>ACTION</b>
+        </td>
+    </tr>
+    <c:forEach items="${cars}" var="car">
+        <tr style="font-size: 25px;">
+            <td style="text-align:center; padding: 15px">
+                <c:out value="${car.id}"/>
+            </td>
+            <td style="text-align:center; padding: 15px">
+                <c:out value="${car.model}"/>
+            </td>
+            <td style="text-align:center; padding: 15px">
+                <c:out value="${car.manufacturer.name}"/>
+            </td>
+            <td style="text-align:center; padding: 15px">
+                <c:forEach items="${car.drivers}" var="driver">
+                    <c:out value="${driver.name}  "/><br>
```

Comment:
maybe also show his id or license number, because two drivers can have the same name